### PR TITLE
TE-8042 / TE-7945 Replaced Travis with GA

### PR DESCRIPTION
**Pull request by Nidhognit**
Task https://spryker.atlassian.net/browse/TE-8042

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+  - cron: "0 0 * * *"
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.2',
```
Comment (dereuromark) [inline .github/workflows/ci.yml:17]: Do we need to support php7.1 or PHP7.2 here still?
If so, that's OK
I would bump
https://github.com/spryker-sdk/codeception-phantoman/blob/master/composer.json#L7
at least to 7.2

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      - name: PHPStan checks
+        run: composer stan
+
+  prefer-lowest:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.1'
```
Comment (dereuromark) [inline .github/workflows/ci.yml]: We use 7.2+, usually already 7.3+ at this time.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+jobs:
+  validation:
+    runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        php-version: [
+            '7.2',
+            '7.3'
```
Comment (dereuromark) [inline .github/workflows/ci.yml]: Use 7.4 here then, also maybe 8.0 already on top as well

File: .github/workflows/ci.yml (hunk 1 above)
Comment (Nidhognit) [reply .github/workflows/ci.yml:17]: I see that composer require >= 7.1, and old travis file supported it, should i update composer file?

Comment (dereuromark) [reply .github/workflows/ci.yml:17]: Yes.72+

File: .github/workflows/ci.yml [hunk 4]
```diff
@@ -58,8 +59,7 @@ jobs:
       fail-fast: false
       matrix:
         php-version: [
-            '7.2',
-            '7.1',
+            '7.2'
```
Comment (dereuromark) [inline .github/workflows/ci.yml:62]: ~~max is missing here~~
Doesnt match the min above.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -14,8 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         php-version: [
-            '7.4',
-            '7.3'
+            '7.2',
+            '7.3',
+            '7.4'
```
Comment (dereuromark) [inline .github/workflows/ci.yml:18]: Same issue 
Only min/max

**Pull request merged**